js/views/modals/Dialog.js

```diff
@@ -45,7 +45,8 @@ export default class extends BaseModal {
       messageClass: '',
       buttons: [],
       removeOnClose: true,
-      defaultBtnClass: 'btn clrP clrBr clrSh2',
+      // defaultBtnClass: 'btn clrP clrBr clrSh2',
+      defaultBtnClass: 'flexExpand btnFlx clrP',
```

Comment:
I think we should still keep the border and shadow color classes: `clrBr clrSh2`

Reply:
We don't want a shadow class here or we'd get the button shadow overlapping the dialog shadow.